src/Microsoft.AspNetCore.Authentication/ClaimsTransformationMiddleware.cs

```diff
@@ -35,17 +36,22 @@ public class ClaimsTransformationMiddleware
 
         public async Task Invoke(HttpContext context)
         {
-            var handler = new ClaimsTransformationHandler(Options.Transformer, context);
+            var transform = Options.Transformer;
+            if (transform == null && Options.TransformerType != null)
```

Comment:
If you "hardcode" the transformer type to `IClaimsTransformer`, one won't be able to use multiple transformers in the same pipeline.

https://github.com/aspnet/Security/issues/863#issuecomment-254304602